src/components/tag-skill/tag-skill.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import iconTick from '../../svgs/icon-tick.svg';
+import Icon from '../icon/icon.jsx';
+import Tag from '../tag/tag.jsx';
+import styles from './tag-skill.css';
+
+export default function TagSkill(props) {
+  return (
+    <Tag readOnly>
```

Comment:
Do we want to assert that it's `readOnly` here? Traditionally we've set read-only in big maven and left these components as they are. I worry about having to deal with removing the `readOnly` when we make custom apps writable.

Reply:
As per design, TagSkill are always going to be readOnly. The editing UX is a table of sorts.